.github/issue_template.md

```diff
@@ -21,7 +21,7 @@
  - `npm -v` prints:
  - `node -v` prints:
  - `npm config get registry` prints:
- - Windows, OS X, or Linux?:
+ - Windows, OS X/macOS, or Linux?:
```

Comment:
If we're gonna do this how about just making it be macOS and ditching the "OS X"?